axelor-account/src/main/resources/views/AccountingSituation.xml

```diff
@@ -251,7 +251,7 @@
   <action-record name="action-record-manage-vat-system-onnew"
     model="com.axelor.apps.account.db.AccountingSituation">
     <field name="vatSystemSelect" expr="eval: 1"
-      if="partner != null &amp;&amp; (partner.isCustomer != null &amp;&amp; partner.isCustomer) || (partner.isSupplier != null &amp;&amp; partner.isSupplier)"/>
+      if="partner?.isCustomer || partner?.isSupplier"/>
```

Comment:
Should use __parent__ if appears on onNew from Partner form.

Reply:
I will create a ticket in AOS sigma